editor/gui/editor_run_bar.h

```diff
@@ -109,6 +111,9 @@ class EditorRunBar : public MarginContainer {
 
 	Button *get_pause_button() { return pause_button; }
 
+	void add_child_control(Control *p_control);
+	void remove_child_control(Control *p_control);
```

Comment:
We don't need `remove_child_control` in C# since the plugin is never disabled. I added it for parity, but I'm fine without it.